falco/templates/pod-template.tpl

```diff
@@ -56,6 +56,10 @@ spec:
         {{- include "falco.securityContext" . | nindent 8 }}
       args:
         - /usr/bin/falco
+        {{- if not .Values.driver.enabled }}
+        - --disable-source
+        - syscall
+        {{- end }}
```

Comment:
We need a helper or something like that because there could be more complex cases to be handled.